network_switch/Makefile

```diff
@@ -0,0 +1,11 @@
+LIBFLAGS=-pthread -lrt -Wall
```

Comment:
Here is the network switch code; all of the folders under `runtime/network_switch` were created on this branch.